pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -94,6 +94,7 @@ stdenv.mkDerivation ({
   patches = [
     ./env_var_for_system_dir.patch
   ]
+  ++ lib.optional waylandSupport ./fedora-pipewire.patch
```

Comment:
will update the filename

Reply:
Could you fetch it from fedora instead of putting it in-tree?

Reply:
@jtojnar I'll update. Please note, this isn't the Fedora patch though (when I took the relevant patch from master, it didn't build, idk if our Firefox is newer?). This patch is here, which I believe was the repo building the Nightly Flatpak that most folks have been using for quite sometime: https://github.com/xhorak/firefox-devedition-flatpak

Reply:
Yeah, use `https://github.com/xhorak/firefox-devedition-flatpak/raw/303388734fb213a9c4e5836f5017a4a849372bdb/org.mozilla.FirefoxNightly/firefox-pipewire.patch` then.